core/domain/stats_domain.py

```diff
@@ -43,28 +43,32 @@ def total_answer_count(self):
         return sum(self.answers.values())
 
     @classmethod
-    def get_multi(cls, exploration_id, rule_data):
-        """Gets domain objects corresponding to the given rule data.
+    def get_multi(cls, exploration_state_list, rule_str_list):
+        """Gets domain objects given a list of exploration and state tuples.
 
         Args:
-            exploration_id: the exploration id
-            rule_data: a list of dicts, each with the following keys:
-                (state_name, rule_str).
+            exploration_state_list: a list of exploration ID and state name
+                tuples
+            rule_str_list: a list of rule spec strings which are used to filter
+                the answers matched to the provided explorations and states
+
+        Returns a list with a list of StateRuleAnswerLog objects for each group
+        of answers matched to each exploration ID-state name.
```

Comment:
Added a sentence clarifying. The number of elements returned should be ``len(exploration_state_list)``